be/src/column/column_access_path.cpp

```diff
@@ -136,4 +136,139 @@ const std::string ColumnAccessPath::to_string() const {
     return ss.str();
 }
 
+void ColumnAccessPathUtil::rewrite_struct_type_descriptor(TypeDescriptor& original_type,
```

Comment:
```suggestion
void ColumnAccessPathUtil::rewrite_struct_type_descriptor(const TypeDescriptor& original_type,
```
use const for reference.